render/pod.go

```diff
@@ -96,6 +96,28 @@ var ReplicaSetRenderer = ConditionalRenderer(renderKubernetesTopologies,
 	),
 )
 
+// DaemonSetRenderer is a Renderer which produces a renderable kubernetes daemonsets
+// graph by merging the pods graph and the daemonsets topology.
+var DaemonSetRenderer = ConditionalRenderer(renderKubernetesTopologies,
+	MakeMap(
+		PropagateSingleMetrics(report.Pod),
+		MakeReduce(
+			MakeMap(
+				Map2Parent(report.DaemonSet, "", nil),
+				PodRenderer,
+			),
+			SelectDaemonSet,
+		),
+	),
+)
+
+// ReplicaGroupRenderer is a Renderer which combines the daemonset and deployment views
+// in (for now) a very naive way.
+var ReplicaGroupRenderer = MakeReduce(
+	DeploymentRenderer,
+	DaemonSetRenderer,
```

Comment:
So, the ReplicaGroup view will also include DaemonSets? Why have a separate view for DaemonSets alone then?

Also, I am not sure I like the name (ReplicaGroup) or the entities it groups. 

I think it would make sense to group controllers together (that is, ReplicaSets, ReplicationControllers, DaemonSets, PetSets) in a view and name it  "Controllers". However, Deployments live one level of abstraction higher from controllers (they are on top of ReplicaSets) which also makes the name confusing.